source/common/http/conn_manager_impl.cc

```diff
@@ -284,6 +284,7 @@ Network::FilterStatus ConnectionManagerImpl::onData(Buffer::Instance& data, bool
     }
   }
 
+  // TODO(alyssawilk) clean this up after #8352 is well vetted.
```

Comment:
Also added this but now I'm wondering if we can land the connection fix or we have to clean this up inline or something weird will happen?  I'll take a look tomorrow.